tests/testthat/test-type2.R

```diff
@@ -158,11 +158,11 @@ test_that("Subclasses of data.frame dispatch to `vec_ptype2()` methods", {
   expect_identical(vec_ptype2(mtcars, quux), "dispatched!")
 })
 
-test_that("Subclasses of `tbl_df` have `tbl_df` common type (#481)", {
+test_that("Subclasses of `tbl_df` do not have `tbl_df` common type (#481)", {
```

Comment:
Maybe update or remove the issue number so we aren't confused if we look back on this?

```suggestion
test_that("Subclasses of `tbl_df` do not have `tbl_df` common type (#710)", {
```